src/lib/ptree/ptree.lua

```diff
@@ -334,6 +334,13 @@ function Manager:make_rrd(counter_name)
       base_interval='2s' })
 end
 
+local function blacklisted (name)
+   local t = lib.set('macaddr', 'mtu', 'promisc', 'speed', 'status', 'type')
+   name = lib.basename(name)
+   name = name:gsub("%.counter", "")
```

Comment:
2 nits to consider:

 - can lift the `lib.set` out of the function, e.g. `local blacklisted_counters = lib.set(...); local function blacklisted(name)...end`

- can use strip_suffix here; e.g.
```
return blacklisted_counters[strip_suffix(lib.basename(name), '.counter')]
```